nodejs/README.adoc

```diff
@@ -0,0 +1,341 @@
+** For the bootstrap server endpoint, select the options menu (three vertical dots) for a Kafka instance in {product}. Click *View connection information*.
+** For the client ID and client secret, you're prompted to copy and store these credentials when you create a service account in {product}.
+endif::[]
+
+.Procedure
+
+. In your IDE, open a new file for the shell script. Save the file with the name `set-environment.sh`.
+
+. In the `set-environment.sh` file, add the lines shown in the example to set the bootstrap server and client credentials as environment variables to be used by the Node.js application.
++
+.Setting environment variables for bootstrap server and client credentials
+[source,subs="+quotes"]
+----
+export KAFKA_BOOTSTRAP_SERVER=__<bootstrap_server>__
+export KAFKA_CLIENT_ID=__<client_id>__
+export KAFKA_CLIENT_SECRET=__<client_secret>__
+export KAFKA_SASL_MECHANISM=plain
+----
++
+In the preceding example, replace the values in angle brackets (`< >`) with your own bootstrap server and client credential information.
+ifdef::qs[]
+These details are described below.
++
+--
+<bootstrap_server>:: The bootstrap server endpoint for your service account. To access this information for a Kafka instance in {product}, select the options menu (three vertical dots). Click *View connection information*.
+<client_id>:: A client credential generated when you create a service account in {product}. You're prompted to copy and store this credential when you create the service account.
+<client_secret>:: A client credential generated when you create a service account in {product}. You're prompted to copy and store this credential when you create the service account.
+--
+endif::[]
++
+In this case, observe that the Node.js application uses the `SASL/PLAIN` authentication method (that is, the value of `KAFKA_SASL_MECHANISM` is set to `plain`). This means that the application uses only the client ID and client secret to authenticate with the Kafka instance. The application *doesn't* also require an authentication token.
+
+. Save the the `set-environment.sh` file.
+
+ifdef::qs[]
+.Verification
+* Did you set environment variables for the Kafka instance?
+endif::[]
+
+[id="proc-creating-countries-topic_{context}"]
+== Creating a Kafka topic in {product}
+
+In this quick start, the Node.js example application requires a Kafka topic called `countries`. You need to create this topic in your Kafka instance so that the application can interact with it.
+
+.Prerequisites
+* You've created a Kafka instance in {product} and the instance is in the *Ready* state.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to add a topic to.
+. Click *Create topic* and follow the guided steps to define the topic details. Click *Next* to complete each step and click *Finish* to complete the setup.
++
+[.screencapture]
+.Guided steps to define topic details
```

Comment:
This repeats a lot with other guides. Worth checking if content is the same

Reply:
The version in the doc I created was copied from the Quarkus guide, except for the topic create.

Reply:
Yes, we are using the same content as for other quick starts (for example, Quarkus), but with some minor tweaks.